### ISSUE TITLE
Overwriting previous delimiting key; Ambiguous paths ... are not official Warning messages

### ISSUE THREAD

**Issue by andrew-appel** [coq/coq#8207]
#### Description of the problem

Run this command (assuming the standard mathcomp installation for Coq 8.8.0):
`Require Import mathcomp.ssreflect.ssrbool.`

and I get these messages:
```
Overwriting previous delimiting key bool in scope bool_scope
Ambiguous paths:
[pred_of_mem_pred; sort_of_simpl_pred] : mem_pred >-> pred_sort
```

The word "Warning" does not appear, and in particular there is no indication of what Set Warning flag controls this message.

That part seems to be a Coq bug.  

For the ssreflect team, I wonder whether it's possible to write a line such as, 
`From mathcomp.ssreflect Require Import ssreflect seq ssrbool ssrnat ssrfun eqtype seq fintype finfun.
`
and not get any warning messages at all.  Am I doing something wrong?

**Comment by tchajed** [comment]
@andrew-appel Thanks for the bug report! I'm always a fan of improving error messages.

These two warnings not using the new configurable warning system is a clear bug, so I submitted PRs to fix those two issues.

However, this doesn't address the fact that ssreflect triggers these warnings. The overwritten scope is bool (from the Coq prelude) being replaced with B in ssrbool and the ambiguous coercions paths are just an artifact of ssreflect's coercions. I'm not sure what ssreflect could do to avoid these, other than locally disabling the warnings since they're probably harmless for these two cases.

**Comment by Zimmi48** [comment]
Yes, we'll probably have to disable these warnings locally since they are a conscious choice. The longer term solution (i.e. in stdlib2) is to be consistent between the choices made in Coq's prelude and in SSR (cc @gares @amahboubi @maximedenes).

**Comment by pi8027** [comment]
I believe that this problem has been fixed in #9743. May I close?

**Comment by gares** [comment]
> For the ssreflect team, I wonder whether it's possible to write a line such as,

As of today there is no way :-/
